NEWS.md

```diff
@@ -1,5 +1,6 @@
 # Version 2.0.0.9158
 * **Note: This release (1.0 -> 2.0) signifies a major breaking revamp of the package.** Users are advised to carefully review the release notes and documentation for detailed information on the changes and any necessary updates to their existing code.
+* Corrected a typo in the example vignette.
```

Comment:
```suggestion
```
(no need to mention this, because we have some many large changes and this is nothing informative for the user)